tests/test_functions.py

```diff
@@ -110,3 +112,71 @@ def test_extract_hashbits_info():
             os.remove(fn)
         except OSError as e:
             print >>sys.stderr, '...failed to remove {fn}'.format(fn)
+
+
+FakeFQRead = collections.namedtuple('Read', ['name', 'accuracy', 'sequence'])
+FakeFastaRead = collections.namedtuple('Read', ['name', 'sequence'])
+
+
+def test_check_is_pair_1():
+    read1 = FakeFQRead(name='seq', accuracy='###', sequence='AAA')
+    read2 = FakeFQRead(name='seq2', accuracy='###', sequence='AAA')
+
+    assert not check_is_pair(read1, read2)
+
+
+def test_check_is_pair_2():
+    read1 = FakeFQRead(name='seq/1', accuracy='###', sequence='AAA')
+    read2 = FakeFQRead(name='seq/2', accuracy='###', sequence='AAA')
+
+    assert check_is_pair(read1, read2)
+
+
+def test_check_is_pair_3():
+    read1 = FakeFQRead(name='seq 1::', accuracy='###', sequence='AAA')
+    read2 = FakeFQRead(name='seq 2::', accuracy='###', sequence='AAA')
+
+    assert check_is_pair(read1, read2)
+
+
+def test_check_is_pair_4():
```

Comment:
I rule this an unnecessary conversation :).  Any changes to function
or syntax that need to be made??